art.py

```diff
@@ -31,7 +31,7 @@ def index_art(song):
             if tag.startswith('APIC'):
                 data = tags[tag].data
                 break
-    elif isinstance(tags, MP4Tags) and tags['covr']:
+    elif isinstance(tags, MP4Tags) and 'covr' in tags.keys() and tags['covr']:
```

Comment:
`'covr' in tags` is preferable here, as it utilizes hashing rather than a linear search.